appscale/tools/parse_args.py

```diff
@@ -375,6 +375,32 @@ def add_allowed_flags(self, function):
       self.parser.add_argument('--keyname', '-k',
         default=self.DEFAULT_KEYNAME,
         help="the keypair name to use")
+    elif function == "appscale-show-stats":
+      self.parser.add_argument('--keyname', '-k',
+        default=self.DEFAULT_KEYNAME,
+        help="the keypair name to use")
+      self.parser.add_argument('--show',
+        nargs='+',
+        choices=['nodes', 'processes', 'proxies'],
+        default=['nodes', 'proxies'],
+        help="the list of entities that statistics should be printed")
```

Comment:
"I would also suggest --type instead of --show, but I don't feel strongly about it."
Is it appropriate in case when an user can specify a few types there?